test/TestGrainInterfaces/TestGrainInterfaces.2017.csproj

```diff
@@ -1,35 +1,27 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net461</TargetFramework>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <RuntimeIdentifier>win</RuntimeIdentifier>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <RootNamespace>UnitTests.GrainInterfaces</RootNamespace>
     <AssemblyName>TestGrainInterfaces</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Remove="IFSharpParametersGrain.cs" />
-    <Compile Remove="IGeneratorTestDerivedFromFSharpInterfaceInExternalAssemblyGrain.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="..\..\src\Build\GlobalAssemblyInfo.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="IFSharpParametersGrain.cs" />
-    <None Include="IGeneratorTestDerivedFromFSharpInterfaceInExternalAssemblyGrain.cs" />
-  </ItemGroup>
-  <ItemGroup>
     <ProjectReference Include="..\..\src\Orleans\Orleans.2017.csproj" />
     <ProjectReference Include="..\..\src\OrleansRuntime\OrleansRuntime.2017.csproj" />
     <ProjectReference Include="..\TestInterfaces\TestInterfaces.2017.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(IncludeFSharp)'=='true'">
+    <ProjectReference Include="..\TestFSharpInterfaces\TestFSharpInterfaces.2017.fsproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="FSharp.Core" Version="4.1.17" />
```

Comment:
hmm, I guess probably not, as I believe we have some tests against FSharp basic types, regardless of whether our FSharp grain projects are being built

---
In reply to: [118301010](https://github.com/dotnet/orleans/pull/3038#discussion_r118301010) [](ancestors = 118301010)